src/test/java/guitests/ContextMenuTests.java

```diff
@@ -97,4 +105,27 @@ public void test3() {
         sleep(EVENT_DELAY);
     }
 
+    /**
+     * Tests marked issues from Issue #9 in the UI right to the end of the list as read/unread
+     * @param isTestingRead If true, tests whether issues on/below the selected issue are being correctly marked read
+     *                      If false, tests whether issues on/below the selected are being correctly marked unread
+     */
+    private void markAllReadOrUnread(boolean isTestingRead){
```

Comment:
Method name doesn't match the intent of the method. 

In fact, I'm not sure if this method is a good unit of abstraction. Methods are not simply containers for a chunk of statements. They are abstractions. Here are some better choices for methods:
`markIssuesBelow(int index, boolean isMarkAsRead)`
`verifyReadStatusOfIssuesBelow(int index, boolean isExpectedStatusRead)`

Using these abstractions, you can do a better job of testing. e.g.
```
take a panel of 10 issues.
mark all below 5.
check if all below 5 are marked as read.
mark all below 5 as read (again).
ensure they are still marked as read.
...
```